data/json/items/containers.json

```diff
@@ -659,7 +665,8 @@
     "color": "green",
     "contains": "10L",
     "seals": true,
-    "watertight": true
+    "watertight": true,
+    "qualities": [ [ "BOIL", 1 ] ]
```

Comment:
Removed `BOIL` from plastic jerrycan. Didn't realize they could be plastic. List of touched `id`s coming soon.